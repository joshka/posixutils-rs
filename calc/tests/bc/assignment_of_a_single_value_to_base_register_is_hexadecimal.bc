obase = F
obase

ibase = 2
ibase = D
ibase


ibase = 13
ibase = F
ibase

quit
