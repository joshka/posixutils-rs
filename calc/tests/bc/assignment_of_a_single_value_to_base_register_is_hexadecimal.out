10
D
10
